lib/index.js

```diff
@@ -13,6 +14,14 @@ KeenCore.on('client', function(client){
     events: [],
     collections: {}
   };
+  
+  if (client.config.respectDoNotTrack) {
+    this.doNotTrack = client.config.respectDoNotTrack;
+  } else if (typeof client.config.optOut !== 'undefined') {
```

Comment:
let's break that - so both "if"s will be separated

lets change the first condition to the one below.
```
if (!client.config.respectDoNotTrack) {
    this.doNotTrack = false;
}

if (typeof client.config.optOut !== 'undefined') {
```